src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.iOSmacOS.cs

```diff
@@ -155,6 +157,19 @@ private static IDisposable InnerCreateLayer(UIElement owner, CALayer parent, Lay
 						CreateImageBrushLayers(area, adjustedArea, parent, sublayers, ref insertionIndex, imgBackground, fillMask);
 					}
 				}
+				else if (background is AcrylicBrush acrylicBrush)
+				{
+					// TODO: react to AlwaysUseFallback changes
+					if (acrylicBrush.AlwaysUseFallback)
+					{
+						Brush.AssignAndObserveBrush(acrylicBrush, color => layer.FillColor = color)
+							.DisposeWith(disposables);
+					}
+					else
+					{
```

Comment:
Codacy found an issue: [Either remove or fill this block of code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)